.asf.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+ 
+# The format of this file is documented at
+# https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features
+
+github:
+  description: "Apache Polaris"
+  homepage: https://polaris.apache.org/
+  labels:
+    - polaris
+    - iceberg
+    - apache
+
+  enabled_merge_buttons:
+    merge: true
+    squash: true
+    rebase: true
+
+  protected_branches:
+    main:
+      required_pull_request_reviews:
+        required_approving_review_count: 1
+
+      required_linear_history: true
+
+  features:
+    wiki: false
+    issues: true
+    projects: false
+  ghp_branch: gh-pages
+  ghp_path: /
+
+notifications:
+  commits:      dev@polaris.apache.org
```

Comment:
Is this what the community wants? Given that most people also get github notifications, most of the communities I've worked in use commits@ instead of dev@ for this.

Reply:
Good catch. Let me create the mailing lists and update there. Thanks !

Reply:
Mailing lists creation request has been submitted (`commits` and `issues`). @snazy created another PR extracting just `.asf.yml` from here. Let's use the other PR, I will clean up in this PR.